doc/howtos/rdtraining/architecture.rst

```diff
@@ -0,0 +1,100 @@
+.. _howto/rdtraining/architecture:
+
+=====================
+Architecture Overview
+=====================
+
+Multitier application
+=====================
+
+Odoo follows a `multitier architecture`_ architecture, meaning that the presentation, the business
+logic and the data storage are separated. More specifically, it uses a three-tier architecture
+(image from Wikipedia):
+
+.. image:: architecture/media/three_tier.svg
+    :align: center
+    :alt: Three-tier architecture
+
+The presentation tier is a combination of HTML5, JavaScript and CSS. The logic tier is exclusively
+written in Python, while the data tier only supports PostgreSQL as an RDBM.
+
+Depending on the scope of your module, the Odoo development can be done in any of these tiers.
+Therefore, before going further it may be a good idea to refresh your memory if you don't have
+an intermediate level in these topics.
+
+In order to go through this tutorial, you will need very basic HTML knowledge but an intermediate
+level of Python. Advanced topics will require more knowledge in the other subjects. There are
+plenty of tutorials freely accessible, so we cannot really recommend one more than another;
+it all depends on your background.
+
+For reference this is the official `Python tutorial`_.
+
+Odoo modules
+============
+
+Both server and client extensions are packaged as *modules* which are
```

Comment:
I'm not sure this phrase is understandable. You can say that a collection of functions that aim a sole propose are packaged as modules. This functions can be any of or a combination of the 3 tier ?